### ISSUE TITLE
Image Preview

### ISSUE THREAD

**Issue by shift-reality** [manusa/isotope-mail#332]
Hi.
Now attachments can be downloaded but there no preview feature available.
It's good practice to have little built-in **gallery of images** in current email.
Please, implement this feature for add more usability to client.
Thanks!

**Comment by mshannaq** [comment]
also while you in inbox there is no notice that the message has attachment also.

**Comment by manusa** [comment]
Image preview is somehow difficult as all server-side tasks are delegated to IMAP server, so this would imply downloading the complete image and then sending a thumbnail version of it (which will imply additional network traffic).

Probably what can be implemented is image preview for small-sized images/documents.

**Comment by shift-reality** [comment]
Sorry, i missed words...
Not "Preview" like "thumb small image"...
**I mean "Preview modal", when you click on attachment in mail this simply start downloading... But need add button "Preview" which shows modal popup with Image and next&prev buttons (like galleries).....**

**Comment by shift-reality** [comment]
"Download" and "View" buttons on attachments

**Comment by mshannaq** [comment]
> also while you in inbox there is no notice that the message has attachment also.

Yes , and it is good to show related icon beside the attachment, for example if a PFD file attached then show pdf icon .... etc.  also showing the size of attached file.
